doc/_static/mpl.css

```
@@ -185,3 +187,11 @@ div.wide-table table th.stub {
 .sdd.sd-dropdown .sd-card-header +.sd-card-body{
     --pst-sd-dropdown-color: none;
 }
+
+/*removes page title from toctree listing*/
+.section-toc.toctree-wrapper .toctree-l1>a{
+    display: None;
```

Doesn't really matter, but CSS usually uses lowercase.

```suggestion
    display: none;
```